frontend-react/cypress.config.js

```diff
@@ -0,0 +1,56 @@
+"use strict";
```

Comment:
[non-blocking] Do we need this as a `.js` file or is it something that comes dynamically in the build from the origin `.ts` file? or maybe we can turn the original file into a normal `.js` file? 🤔 ...just thinking cuz I've been looking at the cypress stuff for a bit

Reply:
This file was added by mistake. Will remove